fixing timezone problem

Eyup-Devop opened the pull request:
## Summary
When Etc/GMT-3 and Etc/GMT-2 timezone used with toDateTime function, query is created as Etc/GMT-11. To solve this problem, the time.Format function must be used differently.

Eyup-Devop commented:
I have signed the agreement.

CLAassistant ***@***.***>, 24 Oca 2024 Çar, 17:22 tarihinde
şunu yazdı:

> [image: CLA assistant check]
> <https://cla-assistant.io/ClickHouse/clickhouse-go?pullRequest=1182>
> Thank you for your submission! We really appreciate it. Like many open
> source projects, we ask that you sign our Contributor License Agreement
> <https://cla-assistant.io/ClickHouse/clickhouse-go?pullRequest=1182>
> before we can accept your contribution.
> You have signed the CLA already but the status is still pending? Let us
> recheck
> <https://cla-assistant.io/check/ClickHouse/clickhouse-go?pullRequest=1182>
> it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClickHouse/clickhouse-go/pull/1182#issuecomment-1908230414>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6NWXGZ7KUWLFVHRM7PB6QDYQEKLZAVCNFSM6AAAAABCI3CZ5WVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMBYGIZTANBRGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

jkaflik commented:
Hi @Eyup-Devop I have added a test case and fixed implementation for this logic: https://github.com/ClickHouse/clickhouse-go/pull/1184

Eyup-Devop commented:
HelloIt is good to hear that is working. I dont need to work with forked repository anymore. Thank youEYÜP SARALKuba Kaflik ***@***.***> şunları yazdı (29 Oca 2024 14:21):﻿
Hi @Eyup-Devop I have added a test case and fixed implementation for this logic: #1184

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
